### PR TITLE
removed process diagram tab and export pdf button for draft feature

### PR DESCRIPTION
**Pull request by abilpraju-aot**
# Issue Tracking

JIRA: 
Issue Type: BUG/
Removed process diagram tab and export pdf button for draft feature
image
image

### CODE REVIEW

**Review by sreehari-aot — CHANGES_REQUESTED** [review]
Do not remove process diagram

File: forms-flow-web/src/components/Draft/ViewDraft.js [hunk 1]
```diff
@@ -93,15 +92,6 @@ const ViewDraft = React.memo(() => {
         >
           <View page="draft-detail" />
         </Tab>
-        <Tab
```
Comment (abhilash-aot) [inline forms-flow-web/src/components/Draft/ViewDraft.js:96]: Keep the process Diagram. That is fine , Inicates which flow will trigger on Submission of drat

**Review by AbijithS-aot — APPROVED** [review]

File: forms-flow-web/src/components/Form/Item/Submission/Item/View.js [hunk 2]
```diff
@@ -54,7 +54,7 @@ const View = React.memo((props) => {
     <div className="container row task-container">
       <div className="main-header">
         <h3 className="task-head"> {form.title}</h3>
-        {showPrintButton && form?._id ? (
+        {(showPrintButton && form?._id && !props.submission.draftDetailStatusCode ) ? (
```
Comment (sreehari-aot) [inline forms-flow-web/src/components/Form/Item/Submission/Item/View.js:57]: Need to update the condition with some other reliable property to check if the item is a draft or not

Comment (abhilash-aot) [reply forms-flow-web/src/components/Form/Item/Submission/Item/View.js:57]: Yup, Draft Id and draft enable env check would be better?

Comment (abilpraju-aot) [reply forms-flow-web/src/components/Form/Item/Submission/Item/View.js:57]: ok

**Pull request closed without merging**